### PR TITLE
Feature/mob 113 signout logic

### PR DESCRIPTION
**Pull request by MPilinkus**
User can login, register and signout

### CODE REVIEW

File: src/app/shared/header/header.component.html [hunk 1]
```diff
@@ -3,9 +3,16 @@
...
-    <div>
-      <app-button class="sign-up" [buttonStyle]="signUpButtonStyle" (click)="openSignupPage.emit()">Sign Up</app-button>
-      <app-button class="login" (click)="openLoginPage.emit()">Login</app-button>
+    <div *ngIf="!signedInUser else UserSignedIn">
+      <app-button class="sign-up" [buttonStyle]="signUpButtonStyle" (click)="openSignupPage()">Sign Up</app-button>
+      <app-button class="login" (click)="openLoginPage()">Login</app-button>
     </div>
+    <ng-template #UserSignedIn>
+      <div class="loggedIn">
+        You are signed in!
```
Comment (Dezash) [inline src/app/shared/header/header.component.html]: Could we display something like "Welcome, UserName" here?

Comment (MPilinkus) [reply src/app/shared/header/header.component.html]: For that I need backend support to return me the username.

Comment (Dezash) [reply src/app/shared/header/header.component.html]: Username and email should be inside the jwt payload, perhaps you could use something like this https://www.npmjs.com/package/jwt-decode ?

Comment (MPilinkus) [reply src/app/shared/header/header.component.html]: Good idea. Got it working.

**Review by Dezash — APPROVED** [review]

**Review by lukasrakauskas — APPROVED** [review]

**Pull request merged**